src/test/java/com/squareup/connect/api/TestCatalog.java

```diff
@@ -0,0 +1,253 @@
+package com.squareup.connect.api;
+
+import com.squareup.connect.ApiException;
+import com.squareup.connect.models.BatchDeleteCatalogObjectsRequest;
+import com.squareup.connect.models.BatchUpsertCatalogObjectsRequest;
+import com.squareup.connect.models.BatchUpsertCatalogObjectsResponse;
+import com.squareup.connect.models.CatalogCategory;
+import com.squareup.connect.models.CatalogDiscount;
+import com.squareup.connect.models.CatalogIdMapping;
+import com.squareup.connect.models.CatalogItem;
+import com.squareup.connect.models.CatalogItemModifierListInfo;
+import com.squareup.connect.models.CatalogItemVariation;
+import com.squareup.connect.models.CatalogModifier;
+import com.squareup.connect.models.CatalogModifierList;
+import com.squareup.connect.models.CatalogObject;
+import com.squareup.connect.models.CatalogObjectBatch;
+import com.squareup.connect.models.CatalogTax;
+import com.squareup.connect.models.ListCatalogResponse;
+import com.squareup.connect.models.Money;
+import com.squareup.connect.models.Money.CurrencyEnum;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+
+/**
+ * Helper class for populating a usable catalog on the server.
+ */
+class TestCatalog implements AutoCloseable {
```

Comment:
I'm on the fence about it since this it creates catalog objects on a real account instead of just exposing static objects.